examples/getting_started/2-Customization.ipynb

```diff
@@ -276,7 +294,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note how HoloViews remembered the Bokeh-specific styles we previously applied to the `spikes` object! This feature allows us to style objects once and then keep that styling as we work, without having to repeat the styles every time we work with that object. Note that even though this styling is associated with the element, it is not actually stored on it, which is mostly an implementation detail but does define a strict separation between what HoloViews considers parts of your data (the Element) and what is part of the \"look\" or the \"view\" of that data (the options associated with the object, but stored separately).  You can learn more about the output utility and the options system in the [User Guide](../user_guide/03-Customizing_Plots.ipynb)."
+    "Note how HoloViews remembered the Bokeh-specific styles we previously applied to the `spikes` object! This feature allows us to style objects once and then keep that styling as we work, without having to repeat the styles every time we work with that object. Note that even though this styling is associated with the element, it is not actually stored on it, which is mostly an implementation detail but does define a strict separation between what HoloViews considers parts of your data (the Element) and what is part of the \"look\" or the \"view\" of that data (the options associated with the object, but stored separately).  \n",
+    "\n",
+    "If we want to reset back to the original syling, we can call `.opts.clear()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spikes.select(milliseconds=(2000,4000)).opts.clear()"
```

Comment:
As `select` itself returns a clone, I think this example is a little confusing... `spikes` itself should still be styled after this.

Reply:
Right; that's deliberate.

Reply:
Otherwise revisiting a cell with `spikes` will show spikes losing its styling, which would be complicated to explain.